velox/functions/prestosql/Probability.h

```diff
@@ -118,6 +119,26 @@ struct CauchyCDFFunction {
   }
 };
 
+template <typename T>
+struct LaplaceCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double location, double scale, double x) {
+    result = std::numeric_limits<double>::quiet_NaN();
```

Comment:
Please remove this line. This result assignment is only valid in specific cases. Please assign over there directly. Else this code is very hard to read.